package-lock.json

```diff
@@ -8969,11 +8969,6 @@
       "resolved": "https://registry.npmjs.org/focus-group/-/focus-group-0.3.1.tgz",
       "integrity": "sha1-4PMu2GsNq91v/OvfiY7LMuR/7c4="
     },
-    "folktale": {
```

Comment:
@JaviAnibarro we shouldn't be updating dependencies in a pure translation pull request- can you remove this? I can get in there if you're uncomfortable with rebasing in git

Reply:
@mhluongo already removed it, did I?

Reply:
That appears to have deleted the whole thing, which would break the build for later developmnent :sweat_smile: 

If you don't mind I'm going to fix this up

Reply:
sure please do